packages/devtools_app/lib/src/memory/memory_tracker_model.dart

```diff
@@ -160,78 +151,142 @@ class TreeTracker {
 
       if (allocationSample != null) {
         for (var stacktrace in allocationSample.stacktraces) {
-          final allocation = TrackerAllocation(stacktrace.timestamp);
+          final allocation = TrackerAllocation(
+            stacktrace.timestamp,
+            stacktrace,
+          );
           classEntry.addChild(allocation);
-          int displayDepth = 0;
-          for (var callStackEntry in stacktrace.stacktrace) {
-            if (allocation.displayAll ||
-                displayDepth++ < allocation.callDepthDisplay) {
-              allocation.addChild(TrackerCall(callStackEntry));
-            }
-          }
-          if (!allocation.displayAll) {
-            allocation.addChild(TrackerMore(stacktrace.stacktrace));
-          }
         }
       }
     }
   }
 
-  /// Remove the "More ..." and show the entire call stack.
-  void expandCallStack(TrackerMore item) {
-    final TrackerAllocation allocation = item.parent;
-
-    // Show the entire call stack for this instance.
-    allocation.displayAll = true;
-
-    // Re-construct the callstack for this instance.
-    final leafNodesCount = allocation.children.length;
-
-    assert(allocation.children.last is TrackerMore);
-    allocation.removeLastChild(); // Remove "More ..."
+  Widget _helpTrackScreen(BuildContext context) {
```

Comment:
rename to AllocationTrackingInstructions and make this a widget class instead of a method